server/src/main/java/io/crate/execution/engine/aggregation/impl/util/NumericValueHolder.java

```diff
@@ -25,4 +25,6 @@
 
 public interface NumericValueHolder {
     BigDecimal value();
+
+    void setValue(BigDecimal value);
 }
```

Comment:
Is this a new change? Let me quickly check comments in https://github.com/crate/crate/pull/11694, I remember we had to do some not nice thing to avoid extra unboxing.

Reply:
Yep, unfortunately it didn't fail locally, let's see if this works, and if it is, please take another look.

Reply:
never mind, it's not related